Classes/Configuration/ExtensionConfiguration.php

```diff
@@ -45,7 +45,7 @@ class ExtensionConfiguration implements SingletonInterface
     public function __construct()
     {
         if (class_exists('\TYPO3\CMS\Core\Configuration\ExtensionConfiguration')) {
-            $this->configuration = GeneralUtility::makeInstance(\TYPO3\CMS\Core\Configuration\ExtensionConfiguration::class)->get('restler');
+            $this->configuration = GeneralUtility::makeInstance('TYPO3\CMS\Core\Configuration\ExtensionConfiguration')->get('restler');
```

Comment:
Why did you change that?

Reply:
Sorry, I thought using the class constant for an non existing class leads to an error in PHP. I reverted my changes.